version.tf

```diff
@@ -2,10 +2,6 @@ terraform {
   required_version = ">= 1.0.0"
   required_providers {
     # Use "greater than or equal to" range in modules
-    ibm = {
-      source  = "ibm-cloud/ibm"
-      version = ">= 1.49.0"
-    }
     logdna = {
```

Comment:
This is required, it is included in the documentation to expose the required version, it also implicitly required by all the modules.

https://github.com/terraform-linters/tflint-ruleset-terraform/blob/v0.2.1/docs/rules/terraform_unused_required_providers.md